standalone/portal-api/src/modules/supplier/dto/request/update-supplier-company-tax-and-revenue.dto.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsBoolean, IsDateString, IsNotEmpty, IsNumberString, IsOptional, IsString } from "class-validator";
+
+export class UpdateSupplierCompanyTaxAndRevenueRequest {
+    
+    @ApiProperty({ example: 'Engelbrecht Ltd' })
+    @IsOptional()
+    @IsString()
+    taxNumber?: string;
+  
+    @ApiProperty({ example: '12-12-2022' })
+    @IsDateString()
+    lastFiscalYear?: Date;
+  
+    @ApiProperty({ example: '1000000000' })
+    @IsNotEmpty()
+    @IsNumberString()
+    totalRevenue: string;
+  
+    @ApiProperty({ example: '1000000000' })
+    @IsNumberString()
+    exportValue?: number;
+  
+
+    @ApiProperty({ example: '1000000000' })
+    @IsNotEmpty()
+    @IsBoolean()
+    audited: boolean;
```

Comment:
```suggestion
    @ApiProperty({ example: 'true' })
    @IsBoolean()
    audited?: boolean;
```

Currently in designs this field is not required in quick application, but i think we may have to iterate over DTO changes to match designs over development period anyway

Reply:
done